tests/virtualization/libvirtlxc_setup.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: system containers images test setup, installing libvirt-lxc
+# Maintainer: Cédric Bosdonnat <cbosdonnat@suse.de>
+
+use base "basetest";
+use testapi;
+use strict;
+
+sub run() {
+    select_console 'root-console';
+
+    # Install libvirt's lxc driver
+    assert_script_run("zypper --non-interactive --gpg-auto-import-keys ref");
+    assert_script_run("zypper --non-interactive in libvirt-daemon-lxc libvirt-client libvirt-daemon-config-network");
+
+    # Make sure libvirtd is up and running with default network
+    script_run("systemctl restart libvirtd");
```

Comment:
Better always use `assert_script_run` where you don't explicitly handle failures.
Also, please try to use always single ticks unless you want to parse variables inside strings.